DXRMissions/DeusEx/Classes/DXRMissionsM15.uc

```diff
@@ -92,61 +100,116 @@ function int InitGoals(int mission, string map)
             }
         }
 
+        return 152;
```

Comment:
I think this should be 154 just for forwards compatibility? same in InitGoalsRev
```suggestion
        return 154;
```

Reply:
I guess just 154 so that if we did goal rando for Entrance or Final, they would be 152 and 153?